pkg/api/controllers/volume.go

```diff
@@ -30,21 +30,23 @@ import (
 	c "github.com/opensds/opensds/pkg/context"
 	"github.com/opensds/opensds/pkg/controller/client"
 	"github.com/opensds/opensds/pkg/db"
+	dock "github.com/opensds/opensds/pkg/dock/client"
 	"github.com/opensds/opensds/pkg/model"
 	pb "github.com/opensds/opensds/pkg/model/proto"
 	. "github.com/opensds/opensds/pkg/utils/config"
 )
 
 func NewVolumePortal() *VolumePortal {
 	return &VolumePortal{
-		CtrClient: client.NewClient(),
+		CtrClient:  client.NewClient(),
```

Comment:
I'm trying to figure out if we can make controller and dock gRPC client as a registerable plugin so that the system can call the specified one automatically at the runtime. But it's only the initial idea, I will have to think the detail and give the feedback.